packages/augur-sdk/src/events.ts

```diff
@@ -11,7 +11,7 @@ class EventNameEmitter extends EventEmitter {
         event,
       );
     });
-    return super.emit(type, eventsWithName);
+    return super.emit(type, ...eventsWithName);
```

Comment:
Curious why this was needed? I believe it'll break the connectors as written. They map over the event data but that's probably a bug. We'll need to retool the connectors to allow more than one argument to a Callback type. Currently the callback type is: 
```ts
export type Callback = (data: any) => void;
```

Reply:
I added it because it is inline with the type definition of the parent's method definition. 

See: https://nodejs.org/api/events.html#events_emitter_emit_eventname_args